README.md

```diff
@@ -100,8 +102,8 @@ Note that the **report** method supports sending the usage for multiple transact
 
 ```javascript
 var trans = [
-              { "app_id": "your application id", "usage": {"hits": 1}},
-              { "app_id": "your application id", "usage": {"hits": 1000}}
+              { "service_id": "your service id", "app_id": "your application id", "usage": {"hits": 1} },
```

Comment:
This is not correct. The service id should not be included in each transaction.
Take a look at how `report` is defined: `report: (service_id, trans, callback)`